docs/guides/custom-controls/how-to-create-attached-properties.md

```diff
@@ -31,7 +31,7 @@ public class DoubleTappedBehav : AvaloniaObject
 {
     static DoubleTappedBehav()
     {
-        CommandProperty.Changed.Subscribe(x => HandleCommandChanged(x.Sender, x.NewValue.GetValueOrDefault<ICommand>()));
+        CommandProperty.Changed.Subscribe(new Avalonia.Reactive.AnonymousObserver<AvaloniaPropertyChangedEventArgs>(x => HandleCommandChanged(x.Sender, x.NewValue.GetValueOrDefault<ICommand>()));
```

Comment:
There always was a better version, if developer don't use reactive extensions:

```suggestion
        CommandProperty.Changed.AddClassHandler<AvaloniaObject>((sender, args) => HandleCommandChanged(sender, args.GetNewValue<ICommand>()));
```

Reply:
Although, in this case we can use Interactive type parameter here, and simplify HandleCommandChanged a little